packages/asset-swapper/src/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -691,7 +691,7 @@ export class ExchangeProxySwapQuoteConsumer implements SwapQuoteConsumerBase {
 
 function slipNonNativeOrders(quote: MarketSellSwapQuote | MarketBuySwapQuote): OptimizedMarketOrder[] {
     const slippage = getMaxQuoteSlippageRate(quote);
-    if (!slippage) {
```

Comment:
What if we just made this `slippage < 1e-4`?

Reply:
@kimpers was able to reproduce this with 0.05 slippage parameter. Original error was reported with 0.005 slippage.